package.json

```
@@ -1,19 +1,37 @@
 {
   "name": "open-insights",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "main": "index.js",
```

Just discovered this while trying to import the library into my prototype branch:

```suggestion
  "main": "dist/index.js",
   "types": "dist/index.d.ts",
```